src/test/java/seedu/address/logic/graph/HopCroftKarpTest.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.graph;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.stream.IntStream;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.SampleGraph;
+
+class HopCroftKarpTest {
+    @Test
+    public void execute_sampleGraph1_allMatched() {
+        BipartiteGraph subjectGraph = SampleGraph.getSampleGraphOne();
+        new HopCroftKarp(subjectGraph).execute();
```

Comment:
I think it is better to break this up into 2 lines, one for initialising the `HopCroftKarp` object and the next line to execute.